packages/clickable/components/clickable.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+
+import { cx } from "emotion";
+import { outline } from "../style";
+
+export interface IClickableProps {
+  /**
+   * Children should be a HTML element.
+   */
+  children: React.ReactElement<HTMLElement>;
+  /**
+   * Action is a event handler for the onClick and onKeypress events
+   */
+  action: (event?: React.SyntheticEvent<HTMLElement>) => void;
+  /**
+   * The tabIndex is passed down and is the same as the native tabIndex
+   */
+  tabIndex?: number | string;
+}
+
+class Clickable extends React.PureComponent<IClickableProps, {}> {
+  public static defaultProps: Partial<IClickableProps> = {
+    tabIndex: -1
+  };
+
+  constructor(_) {
```

Comment:
can you give some explanation why you decided to use `_` as a parameter name here instead of something better understandable?